packages/planetmint/src/index.ts

```diff
@@ -25,6 +21,11 @@ import {
   BROADCAST,
   RETRIEVE,
 } from "./actions";
+import * as IPFS from 'ipfs-core'
+import all from 'it-all'
```

Comment:
Ahahaha don´t know if I like a new dependency just for a five line iterator:
https://github.com/achingbrain/it/blob/master/packages/it-all/src/index.ts

Reply:
You are right, I hadn't looked at what it actually did